h/h_api/exceptions.py

```diff
@@ -63,6 +63,12 @@ def __init__(self, data_type, reference):
         )
 
 
+class UnsupportedOperationError(SimpleJSONAPIError):
```

Comment:
Used to say we don't support a given action data type combo like "create group". The syntax is valid, but it's not something we can process.